setup.py

```diff
@@ -55,6 +55,8 @@
 
 # Note: cocotb is not installed properly yet and is missing dependencies and binaries
 # We can still import other files next to setup.py, as long as they're in MANIFEST.in
```

Comment:
Based on your other comment, the remark about `MANIFEST.in` is now false?

Reply:
It's kind of both. Without MANIFEST.in it's not in the sdist, without the second line you can't build it from source or sdist.